src/main/scala/pro/civitaspo/digdag/plugin/ecs_task/run/EcsTaskRunOperator.scala

```diff
@@ -52,6 +53,8 @@ class EcsTaskRunOperator(operatorName: String, context: OperatorContext, systemC
       subTask.set("cluster", cluster)
       subTask.set("tasks", "${last_ecs_task_run.task_arns}")
       subTask.set("timeout", timeout.toString)
+      if (pollingStrategy.isPresent) subTask.set("polling_strategy", pollingStrategy.get())
+      subTask
```

Comment:
```suggestion
```

This line is not needed.

Reply:
It is necessary because the last one is an `if` statement and we need to return back the config

Reply:
no. This functions returns the return value of `withDefaultSubTask` function, not the last line of this block, and the `withDefaultSubTask` returns `subTask` that you can see L79.

Reply:
<img width="1242" alt="Screen Shot 2022-07-13 at 14 32 37" src="https://user-images.githubusercontent.com/97092893/178676874-36aab1a1-37bc-46fc-a023-d2d7ace5e002.png">
`ecsTaskWaitSubTask` need to return a `Config`. My IDE won't let me do it.

Reply:
Ah.. I'm sorry. I was wrong. I defined the block as `f: Config => Config` on L68. You are right. But the return value of this block is not used, so could you change the `f: Config => Config` on L68 to `f: Config => Unit`?